.github/workflows/python-api-client.yaml

```diff
@@ -49,25 +57,44 @@ jobs:
     - name: Setup Ruby
       uses: ruby/setup-ruby@v1
       with:
-        working-directory: clients/python
+        working-directory: clients
         ruby-version: '2.7'
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
 
     - name: Build release as latest
-      working-directory: clients/python
+      working-directory: clients/python-legacy
       run: bundle exec jekyll build -d _site -b /
 
     - name: Build release ${{ steps.docver.outputs.tag }}
-      working-directory: clients/python
+      working-directory: clients/python-legacy
       run: bundle exec jekyll build -d _site/${{ steps.docver.outputs.tag }} -b /${{ steps.docver.outputs.tag }}
 
     - name: Publish to docs repository
       uses: dmnemec/copy_file_to_another_repo_action@v1.1.1
       env:
         API_TOKEN_GITHUB: ${{ secrets.PERSONAL_TOKEN }}
       with:
-        source_file: clients/python/_site/.
+        source_file: clients/python-legacy/_site/.
         destination_repo: treeverse/docs-lakefs-python
         destination_folder: /
         user_email: 'support@treeverse.io'
         user_name: 'python-docs-action'
+
+#    - name: Build release as latest
+#      working-directory: clients/python
+#      run: bundle exec jekyll build -d _site -b /
+
+#    - name: Build release ${{ steps.docver.outputs.tag }}
+#      working-directory: clients/python
+#      run: bundle exec jekyll build -d _site/${{ steps.docver.outputs.tag }} -b /${{ steps.docver.outputs.tag }}
+
+#    - name: Publish to docs repository
+#      uses: dmnemec/copy_file_to_another_repo_action@v1.1.1
+#      env:
+#        API_TOKEN_GITHUB: ${{ secrets.PERSONAL_TOKEN }}
+#      with:
+#        source_file: clients/python/_site/.
+#        destination_repo: treeverse/docs-lakefs-sdk-python
+#        destination_folder: /
+#        user_email: 'support@treeverse.io'
+#        user_name: 'python-docs-action'
```

Comment:
?

Reply:
Disabled the creation of the new SDK until after I test it over TestPypi